### PR TITLE
Feature/trans method

### PR DESCRIPTION
**Pull request by fgrassals**

### CODE REVIEW

File: src/lang.js [hunk 1]
```diff
@@ -63,7 +63,7 @@
             return key;
         }
 
-        var message = this._getMessage(key);
+        var message = this._getMessage(key, replacements);
```
Comment (rmariuzzo) [inline src/lang.js]: @fgrassals the parameter `replacements` is unused here, can you remove and push again?

**Comment by rmariuzzo** [comment]
@fgrassals all looks good! I have just made a comment, please check it so I can merge into develop. I highly appreciate your willingness and effort.

**Comment by fgrassals** [comment]
Done :)

**Comment by rmariuzzo** [comment]
Thanks @fgrassals! I realised it was my fault reverting something in the past. But all good now! :+1:

**Pull request merged**